docs-core/src/main/java/com/sismics/docs/core/dao/ScoreDao.java

```diff
@@ -32,4 +37,35 @@ public String create(Score score, String userId) {
         
         return score.getId();
     }
+
+    /**
+     * Get all scores on a document.
+     * 
+     * @param documentId Document ID
+     * @return List of scores
+     */
+    public List<ScoreDto> getByDocumentId(String documentId) {
+        EntityManager em = ThreadLocalContext.get().getEntityManager();
+        StringBuilder sb = new StringBuilder("select s.SCORE_ID_C, s.SCORE_SKILLS_C, s.SCORE_EXPERIENCE_C, s.SCORE_TRANSCRIPTGPA_C, SCORE_MATCH_C, SCORE_CREATEDATE_D from T_SCORE c");
```

Comment:
Please fix the typo at the end (should be "s"), and add the initial aliases to all the columns.

Reply:
Fixed in the recent commit!